### PR TITLE
A few supply additions and minor changes

### PR DESCRIPTION
**Pull request by Viz-Titan**
More stuff to buy at cargo. Spare IDs, telecommunications, donk pockets
Surgical kit can now store the advanced circular saw and dropper.
Inaprovaline pill bottles can be found in first aid wall lockers. 
CSECO locker will have a combat security belt

### CODE REVIEW

**Comment by Viz-Titan** [comment]
It would've been only 2 commits but I screwed it up a bit. Should still be good, though needs review on the changes for approval.

File: code/datums/supplypacks/engineering.dm [hunk 1]
```diff
@@ -195,16 +195,50 @@
...
+// /decl/hierarchy/supply_pack/engineering/bluespacerelay
+// 	name = "Emergency Bluespace Relay Assembly Kit"
+// 	contains = list(/obj/item/circuitboard/bluespacerelay,
+// 					/obj/item/stock_parts/manipulator,
+// 					/obj/item/stock_parts/manipulator,
+// 					/obj/item/stock_parts/subspace/filter,
+// 					/obj/item/stock_parts/subspace/crystal,
+// 					/obj/item/storage/toolbox/electrical)
+// 	cost = 75
+// 	containername = "\improper emergency bluespace relay assembly kit"
```
Comment (DTraitor) [inline code/datums/supplypacks/engineering.dm:207]: Just remove it

Comment (DTraitor) [inline code/datums/supplypacks/engineering.dm]: Just remove it

File: code/datums/supplypacks/hospitality.dm [hunk 2]
```diff
@@ -57,6 +57,15 @@
...
 
+/decl/hierarchy/supply_pack/hospitality/donkpocket
+	name = "Donk pocket crate"
+	contains = list(
+			/obj/item/storage/box/donkpockets,
+			/obj/item/storage/box/donkpockets,
+			/obj/item/storage/box/donkpockets
+			)
+	cost = 10
+	containername = "\improper crate of donk pockets"
```
Comment (DTraitor) [inline code/datums/supplypacks/hospitality.dm]: You forgot to capitalize the name

**Review by DTraitor — CHANGES_REQUESTED** [review]
Just some tweaks

**Comment by Viz-Titan** [comment]
Did the changes

**Review by DTraitor — APPROVED** [review]

**Pull request merged**